app/javascript/src/components/Dashboard/Contacts/Pane/CreateContact.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui/v2";
+
+import Form from "./Form";
+
+export default function NewContactPane({ showPane, setShowPane }) {
```

Comment:
Please rename the component and update the file structure based on previous reviews. 
Also, if you do want to do component splitting here, please create a folder called `Create` inside of which we have `index.jsx` that deals with all the Pane related logic and `Form.jsx` that deals with all form related logic.